app/ui-react/packages/ui/src/Layout/AppLayout.tsx

```diff
@@ -77,9 +78,22 @@ export const AppLayout: React.FunctionComponent<ILayoutBase> = ({
   const onNavToggleMobile = () => {
     setIsNavOpenMobile(!isNavOpenMobile);
   };
+  const [curViewportWidth, setCurViewportWidth] = React.useState(1024);
   const onPageResize = (props: { mobileView: boolean; windowSize: number }) => {
     setIsMobileView(props.mobileView);
+    setCurViewportWidth(props.windowSize);
   };
+  const [isTabletView, setIsTabletView] = React.useState(false);
+  const LARGE_VIEWPORT_BREAKPOINT = parseInt(
+    global_breakpoint_lg.value.substring(
+      0,
+      global_breakpoint_lg.value.indexOf('px')
+    ),
+    10
+  );
+  React.useEffect(() => {
+    setIsTabletView(curViewportWidth <= LARGE_VIEWPORT_BREAKPOINT);
+  }, [curViewportWidth]);
```

Comment:
You're missing the `setIsTabletView` function in the list of dependencies

Reply:
Not sure I understand, what do you mean missing? It's a local setter that doesn't need to be driven based on anything other than the viewport size.. as far as I can tell anyway. Is there a bug I'm not seeing?

Reply:
Your effect depends on `setIsTabletView`, so it should be declared in the list of dependencies together with `curViewportWidth`.

It's a _potential source_ of bug. In this case it wouldn't matter, but better start making an eye for these things. We don't have [this](https://github.com/Gelio/tslint-react-hooks) for the packages yet, only for the main app; that would have warned you about this.